cmake/boards.cmake

```diff
@@ -95,18 +95,17 @@ if(CMAKE_SCRIPT_MODE_FILE AND NOT CMAKE_PARENT_LIST_FILE)
 # some other script
 
 # The options available are:
-# BOARD_ROOT_SPACE_SEPARATED: Space-separated board roots
+# BOARD_ROOT: Semi-colon separated board roots
```

Comment:
Changed so that `boards.cmake` so that user can specify `BOARD_ROOT` in same way as they do when building an application:
https://docs.zephyrproject.org/latest/application/index.html?highlight=board_root#boards